src/Parser/Rogue/Subtlety/Modules/Features/ShadowDance.js

```diff
@@ -41,6 +50,23 @@ class ShadowDance extends Analyzer {
       this.inShadowDance = false;
     }
   }
+
+  statistic() {
+    const shadowBladesUptime = this.combatants.selected.getBuffUptime(SPELLS.SHADOW_BLADES.id) / this.owner.fightDuration;
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.SHADOW_BLADES.id} />}
+        value={`${formatPercentage(shadowBladesUptime)} %`}
+        label={(
+          <dfn data-tip={'Shadow Blades up time'}>
+            Shadow Blades up time
```

Comment:
Only include tooltips if they add new information (which this one does not). Also, 'uptime' is one word.

Reply:
fixed